src/editor.c

```diff
@@ -2045,8 +2114,13 @@ gboolean editor_show_calltip(GeanyEditor *editor, gint pos)
 	editor_find_current_word(editor, pos - 1, word, sizeof word, NULL);
 	if (word[0] == '\0')
 		return FALSE;
-
-	str = find_calltip(word, editor->document->file_type);
+	
+	scope[0] = '\0';
+	//const gchar *sname;
+	//symbols_get_current_scope(editor->document, &sname);
```

Comment:
I would have like to use symbols_get_current_scope here, but it kept returning "unknown" if there is a fix so I can remove read_scope_prefix function then please let me know